dev/conductor/core/lib/src/start.dart

```diff
@@ -235,7 +235,7 @@ class StartContext extends Context {
   }) : git = Git(processManager),
   engine = EngineRepository(
     checkouts,
-    initialRef: candidateBranch,
+    initialRef: 'upstream/$candidateBranch',
```

Comment:
Will we get an error during the checkout if the branch already exists in the fork?

Reply:
By namespacing by the remote name, git will disregard the fork's branches. if you *only* provide the branch name, git will check all remotes and find which has the branch in question.

Reply:
Awesome, thanks for the explanation.